resurfemg/helper_functions.py

```diff
@@ -1389,12 +1393,19 @@ def find_peak_in_breath(
             mode='interp',
             cval=0.0,
         )
-        max_ind = (new_array.argmax())
+        max_ind = (new_array2.argmax())
+        max_val = new_array[max_ind]
+        smooth_max = new_array2[max_ind]
+    elif smooth_algorithm == 'convy':
+        new_array2 = running_smoother(new_array)
+        max_ind = (new_array2.argmax())
         max_val = new_array[max_ind]
+        smooth_max = new_array2[max_ind]
     else:
         max_ind = (new_array.argmax())
         max_val = new_array[max_ind]
-    return (max_ind, max_val)
+        smooth_max = max_val
+    return (max_ind, max_val, smooth_max)
```

Comment:
In case a smoothing process that uses an interpolation is applied, the time and amplitude might be altered with respect to the input signal. This means that as a `max_val` a very low value could be returned. This has to be made very clear to the user probably.

Reply:
OK, will update the documentation